crates/stackable-versioned-macros/src/codegen/common/mod.rs

```diff
@@ -58,18 +58,26 @@ pub(crate) fn format_container_from_ident(ident: &Ident) -> Ident {
 ///
 /// See [`DEPRECATED_FIELD_PREFIX`].
 pub(crate) fn remove_deprecated_field_prefix(ident: &Ident) -> Ident {
-    remove_ident_prefix(ident, DEPRECATED_FIELD_PREFIX)
+    format_ident!(
+        "{}",
+        ident
```

Comment:
**nit (non-blocker):** If possible, can we do this (per style guide)?

```suggestion
        "{ident}",
```

Reply:
Changed in 828200e